controllers/storagecluster/storageclasses.go

```diff
@@ -281,6 +283,7 @@ func newCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster) S
 	persistentVolumeReclaimDelete := corev1.PersistentVolumeReclaimDelete
 	allowVolumeExpansion := true
 	managementSpec := initData.Spec.ManagedResources.CephBlockPools
+	disableKeyRotation := !util.IsAnnotationTruthy(initData, keyRotationEnableAnnotation)
```

Comment:
I believe we should use the variable name 'isKeyRotationEnabled'. What do you think?

Reply:
I decided to go with `disableKeyRotation` as it conveys the intent more clearly. If we go with enable we will have to reverse the checks.

The goal is to add the annotation on SC only when KR is disabled.

WDYT?

Reply:
What I mean here is something like this:
	isKeyRotationEnabled := util.IsAnnotationTruthy(initData, keyRotationEnableAnnotation)
Later you do the logic based on this var, 
The main reason to use the value as is in the variable and perform the logic accordingly. Both ways are the same, but it's your choice which one you prefer.